tests/x11/multi_users_dm.pm

```diff
@@ -22,7 +22,7 @@ use utils;
 sub ensure_multi_user_target {
     type_string "systemctl isolate multi-user.target\n";
     reset_consoles;
-    wait_still_screen 10;
+    wait_still_screen 20;
```

Comment:
You really need to wait for a period in time when the screen does not change for 20 seconds? This is not the normal timeout value you should expect. It's stilltime

Reply:
That's a hack I used in local run. Nice catch, let me revert this.